### PR TITLE
space cleaner and fuel dispensers

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR

Adds two reagent tanks from ss13 for janitor and chemistry needs: 
- space cleaner dispenser
- fuel dispenser.

Currently the only ways to clean dirty tiles are spray em or pry and replace and most of janis dont bother to do that. So, i believe adding space cleaner dispenser will encourage them to use sprays more often.

 

## Media

image_2023-11-12_16-31-42

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Tunguso4ka
- added: NT scientists constructed a new reagent tanks for janitor and chemistry: space cleaner dispenser and fuel dispenser.

### CODE REVIEW

**Comment by GoodWheatley** [comment]
The space cleaner makes sense, but why the oil dispenser?

**Comment by Tunguso4ka** [comment]
> The space cleaner makes sense, but why the oil dispenser?

oh, its should be fuel. Sorry me.

**Comment by Tunguso4ka** [comment]
should be all. Fixed description, fixed meta.json. Fixed reagent.

**Comment by brainfood1183** [comment]
couldnt you just fill a tank with the liquid?

**Comment by Tunguso4ka** [comment]
> couldnt you just fill a tank with the liquid?

yes, but i like the idea of having a tank on the wall, plus it has its own pluses like not taking the whole tile.
and simple solutions created by and for simple/smart people, I'm neither of them.

**Comment by Emisse** [comment]
Im fine with the space cleaner one, the fuel one I dont want mapped in chem but we might able to find another use for it

**Comment by UbaserB** [comment]
Sounds useful

File: Resources/Prototypes/Entities/Structures/Wallmounts/walldispenser.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+- type: entity
+  id: CleanerDispenser
+  name: space cleaner dispenser
+  description: Wallmount reagent dispenser.
+  components:
+  - type: WallMount
```
Comment (Emisse) [inline Resources/Prototypes/Entities/Structures/Wallmounts/walldispenser.yml:6]:
```suggestion
  - type: WallMount
      arc: 180
```

Comment (Emisse) [reply Resources/Prototypes/Entities/Structures/Wallmounts/walldispenser.yml:6]: so its not accessible through walls

**Comment by Tunguso4ka** [comment]
oh test fail :(

**Comment by Emisse** [comment]
you need to fix the spacing on it, github spacing made it wonky lol

**Pull request merged**